src/iohumdrum.cpp

```diff
@@ -5479,21 +5517,11 @@ void HumdrumInput::addTrill(hum::HTp token)
     if (hasaccid) {
         accidval = stoi(accid);
         switch (accidval) {
-            case -1:
-                trill->SetAccidupper(ACCIDENTAL_EXPLICIT_f);
-                break;
-            case 0:
-                trill->SetAccidupper(ACCIDENTAL_EXPLICIT_n);
-                break;
-            case 1:
-                trill->SetAccidupper(ACCIDENTAL_EXPLICIT_s);
-                break;
-            case -2:
-                trill->SetAccidupper(ACCIDENTAL_EXPLICIT_ff);
-                break;
-            case 2:
-                trill->SetAccidupper(ACCIDENTAL_EXPLICIT_x);
-                break;
+            case -1: trill->SetAccidupper(ACCIDENTAL_EXPLICIT_f); break;
+            case 0: trill->SetAccidupper(ACCIDENTAL_EXPLICIT_n); break;
+            case 1: trill->SetAccidupper(ACCIDENTAL_EXPLICIT_s); break;
+            case -2: trill->SetAccidupper(ACCIDENTAL_EXPLICIT_ff); break;
+            case 2: trill->SetAccidupper(ACCIDENTAL_EXPLICIT_x); break;
```

Comment:
Did you do the compressed lines, or did clang-format?  Clang-format gave me the less-compressed version.